skopt/tests/test_searchcv.py

```diff
@@ -5,13 +5,16 @@
 import pytest
 import time
 
+import numpy as np
+
 from sklearn.utils.testing import assert_greater
 from sklearn.datasets import load_iris, make_classification
 from sklearn.model_selection import train_test_split
 from sklearn.pipeline import Pipeline
 from sklearn.svm import SVC, LinearSVC
 from sklearn.tree import DecisionTreeClassifier
 from sklearn.base import clone
+from sklearn.base import BaseEstimator, ClassifierMixin
```

Comment:
Where these imports missing before or why did they get added?

Reply:
That was actually for a custom classifier :) Did not get to it yesterday